backend/app/rest/api/rest_public.go

```diff
@@ -386,6 +387,67 @@ func (s *public) telegramQrCtrl(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GET /search?site=siteID&query=query&limit=20&skip=10&sort=[-]field - search documents
+// site - site ID
+// query - user-provided search query
+// limit - number of documents to return
+// skip - number of documents to skip
+// sort - sort by specified field, can be prefixed with "-" to reverse the order
+func (s *public) searchQueryCtrl(w http.ResponseWriter, r *http.Request) {
+	validateAndGetParam := func(value *int, from, to, defaultVal int, name string) bool {
+		if v, err := strconv.Atoi(r.URL.Query().Get(name)); err == nil {
```

Comment:
generally I preffer to avoid else branches if possible. Her setting defaultVal prior to the if will achieve the same result but without else thing